### PR TITLE
👌 PERFORMANCE: big refactor to start getting start:dev auto-loader working

### PR DESCRIPTION
**Pull request by itsbrex**

### CODE REVIEW

File: develop/start/config/browserSwitch.js [hunk 1]
```diff
@@ -18,14 +18,14 @@ module.exports = function browserSwitch(projectDir, browserVendor) {
...
+          port: 8082
+        }).apply(compiler)
       }
     }
   }
 
-  // TODO: Should default to user browser, not Chrome
-  return new RunChromeExtension({extensionPath: projectDir})
+  // Default to user browser
+  return browserVendor
```
Comment (cezaraugusto) [inline develop/start/config/browserSwitch.js:30]: can you return Chrome here? at this point I can't support all browsers

```suggestion
  // TODO: Should default to user browser, not Chrome
  return new RunChromeExtension({extensionPath: projectDir})
```

**Review by cezaraugusto — CHANGES_REQUESTED** [review]
hi @itsbrex! Thanks for the contribution! extension-create is my favorite project, glad you found it useful.

Left a few comments, mostly to keep Chrome as default and remove the popup template which isn't supported.

**Comment by cezaraugusto** [comment]
hi @itsbrex thanks for contributing!

I made significant changes in the project these days, so I'll have to close this one. The good news is that the project is working for Edge again. You can try `npx extension-create dev <path-to-your-extension>` and things should work as expected.

Would be great to have more contributions from you in the future. The project has many moving parts and expert help is always appreciated. Thanks!

**Comment by itsbrex** [comment]
@cezaraugusto so sorry for ghosting you here! Missed this notification. I need to turn off renovate bot notifications so I stop missing *actual* notifications. 🤦‍♂️

**Pull request closed without merging**